glitter/pages/templates/admin/pages/page/change_form.html

```diff
@@ -0,0 +1,98 @@
+{% extends "admin/base_site.html" %}
```

Comment:
Could we extend from `admin/change_form.html` instead and just have `{% block object-tools-items %}`?

Would reduce a lot of code!